scripts/pcm-dpc/covid-19/preprocess.py

```diff
@@ -0,0 +1,180 @@
+        TEMPLATE = ('Node: E:pcm-dpc->E{index}\n'
+                    'typeOf: dcs:StatVarObservation\n'
+                    'variableMeasured: dcs:{SVFullName}\n'
+                    'observationAbout: {geoNode}\n'
+                    'observationDate: C:pcm-dpc->Date\n'
+                    'value: C:pcm-dpc->{SVname}\n\n')
+        idx = 1
+        with open(self.name + '.tmcf', 'a') as f_out:
+            for statvar in self.stat_vars:
+                if statvar == 'CumulativeTestsPerformed':
+                    sv_full_name = 'cumulativeCount_MedicalTest_COVID_19'
+                elif statvar == 'CumulativeTestedPeople':
+                    sv_full_name = 'cumulativeCount_Person_COVID_19_Tested'
+                elif 'Cumulative' in statvar:
+                    sv_full_name = ('cumulativeCount_MedicalConditionIncident_'
+                                    'COVID_19_'+ statvar.replace('Cumulative', ''))
+                elif 'Incremental' in statvar:
+                    sv_full_name = ('incrementalCount_MedicalConditionIncident_'
+                                    'COVID_19_' + statvar)
+                else:
+                    sv_full_name = ('count_MedicalConditionIncident_COVID_19_'
+                                    + statvar)
+                f_out.write(TEMPLATE.format_map({'index': idx,
+                    'SVname': statvar, 'SVFullName': sv_full_name,
+                    'geoNode': geo_node}))
+                idx += 1
+
+    def _translate(self):
+        """translate coloumn names from italian to english"""
+        it2en = {'data': 'Date', 'stato': 'State',
+                 'codice_regione':'RegionCode',
+                 'denominazione_regione': 'RegionName',
+                 'codice_provincia': 'ProvinceCode',
+                 'denominazione_provincia': 'ProvinceName',
+                 'sigla_provincia': 'ProvinceAbbreviation',
+                 'lat': 'Latitude', 'long': 'Longitude',
+                 'ricoverati_con_sintomi': 'HospitalizedsWithSymptoms',
+                 'terapia_intensiva': 'IntensiveCare',
+                 'totale_ospedalizzati': 'Hospitalized',
+                 'isolamento_domiciliare': 'PeopleInHomeIsolation',
+                 'totale_positivi': 'ActiveCase',
+                 'variazione_totale_positivi': 'IncrementalPositiveCase',
+                 'nuovi_positivi': 'IncrementalActiveCase',
+                 'dimessi_guariti': 'CumulativeRecovered',
+                 'deceduti': 'CumulativeDeath',
+                 'totale_casi':'CumulativePositiveCase',
+                 'tamponi': 'CumulativeTestsPerformed',
+                 'casi_testati': 'CumulativeTestedPeople',
+                 'casi_da_sospetto_diagnostico': 'PositiveCasesFromClinicActivity',
+                 'casi_da_screening': 'PositiveCasesFromSurveyAndTest'}
+        self.data = self.data.rename(columns=it2en)
+
+    def set_location(self):
+        pass
+
+    def geo_template(self):
+        pass
```

Comment:
Can do raise NotImplementedError instead of pass

Reply:
Done